libs/tools/builders/elements/schema.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "http://json-schema.org/schema",
+  "id": "packager",
```

Comment:
this id should be the same like the name in the `builders.json` for example `package-elements`